assets/css/sensei-theme-blocks.scss

```diff
@@ -1,6 +1,13 @@
 @mixin text-gray() {
 	opacity: 0.7;
 }
+
+// It's a hack to allow including this file in the editor canvas.
+// See https://github.com/WordPress/gutenberg/blob/e54a9ec2f97e28db3293628e921e40e81cfe5f76/packages/block-editor/src/components/iframe/index.js#L47.
+.wp-block-hack-to-include-css-in-canvas {
```

Comment:
I don't think we need to resort to this, and it relies on the compatibility function in there. 

The new correct way to load styles for the editor is  `add_editor_style` for theme styles, and [`style`, `editorStyle` options for blocks (#4720)](https://github.com/Automattic/sensei/issues/4720), instead of `wp_enqueue_style` - created that issue to update our other blocks too.

Not sure why it wasn't working since we have `'style' => 'sensei-theme-blocks'` in the course navigation block, maybe that needs to `editorStyle` too? Anyway it would need to be added to the other blocks too.

For now it might be easier to just leave the theme blocks styles in `learning-mode.css` (covering the frontend, site editor and lesson template editor), and use the `enqueue_style` for the widget editor. It will be loaded twice for the site editors but it's a tiny file.

Reply:
Changed here for now: https://github.com/Automattic/sensei/pull/4806/commits/de3d2109d5dd775fcf02b18a7a89c0f394c1ed6a

Reply:
I investigated a little more and realized that we weren't registering the blocks properly in the backend. It was fixed here: https://github.com/Automattic/sensei/pull/4806/commits/21d8271b6cdfa399910182204308a9be9c6204c1